### ISSUE TITLE
Fails if $EDITOR has command line arguments

### ISSUE THREAD

**Issue by khaled** [rajatjindal/kubectl-modify-secret#8]
My `EDITOR` environment variable is set to `"code --wait"`

When I run `kubectl modify-secret` I get the following output:

```
Error: exec: "code --wait": executable file not found in $PATH
```

`kubectl edit` works as expected.

BTW, awesome plugin!

**Comment by rajatjindal** [comment]
thank you for your kind words and for reporting this ticket. I've submitted the PR for the same.

could you please try that fix, and let me know if works fine for you.

https://github.com/rajatjindal/kubectl-modify-secret/pull/9

**Comment by khaled** [comment]
@rajatjindal Thanks for the fix! I'll give a try and report back :)

**Comment by khaled** [comment]
@rajatjindal Tested out and it worked.  Thanks again for the fix!

BTW, it might be handy to have a script that generates a manifest for local use.  I tested locally using [these instructions](https://krew.sigs.k8s.io/docs/developer-guide/testing-locally/), but I had to hand edit the `.krew.yaml` manifest, plugging in both the `sha256` and `uri` attributes to be able to install.

**Comment by rajatjindal** [comment]
Thanks for testing it out, i will checkout the instructions to allow making it easy to test.

And i will cut a new release as soon as i have access to laptop. (Apparently github app does not allow to create new release from ios app)

**Comment by khaled** [comment]
Excellent @rajatjindal, thanks!

**Comment by rajatjindal** [comment]
published v0.0.35 with the fix.
